src/models/overdraft.py

```diff
@@ -28,7 +28,7 @@ def from_objects(
         cls, transfer: Transfer, slippage: SolverSlippage, period: AccountingPeriod
     ) -> Overdraft:
         """Constructs an overdraft instance based on Transfer & Slippage"""
-        assert transfer.solver == slippage.solver_address
+        assert transfer.recipient == slippage.solver_address
         assert transfer.token_type == TokenType.NATIVE
         overdraft = transfer.amount_wei + slippage.amount_wei
         assert overdraft < 0, "This is why we are here."
```

Comment:
You have `f"Overdraft(solver={self.account} ({self.name}),"` in this file. Shouldn't that be `f"Overdraft(recipient={self.account} ({self.name}),"`

Reply:
Overdraft is a solver specific object. It is the solver who has over draft as a result of when solver slippage exceeds the collection of outgoing transfers to them. I don't believe this should be changed.